docs/build/substrate-evm.md

```diff
@@ -11,6 +11,8 @@ We provide a custom data source processor for [Parity's Frontier EVM](https://gi
 | Moonbase Alpha | `wss://moonbeam-alpha.api.onfinality.io/public-ws` | `https://api.subquery.network/sq/subquery/moonbase-alpha-dictionary`    |
 | Astar          | `wss://astar.api.onfinality.io/public-ws`          | `https://explorer.subquery.network/subquery/subquery/astar-dictionary`  |
 | Shiden         | `wss://shiden.api.onfinality.io/public-ws`         | `https://explorer.subquery.network/subquery/subquery/shiden-dictionary` |
+| Acala          | `wss://acala-polkadot.api.onfinality.io/public-ws` | Acala Dictionary doesn't currently support EVM                          |
```

Comment:
When can we expect dictionary support?

Reply:
https://github.com/subquery/subql-dictionary/pull/15/

Its already been deployed for mandala https://project.subquery.network/project/subquery/acala-mandala-tc7-dictionary